stdlib/3/hashlib.pyi

```diff
@@ -21,12 +21,12 @@ class _Hash(object):
     def hexdigest(self) -> str: ...
     def update(self, arg: _DataType) -> None: ...
 
-def md5(arg: _DataType = ...) -> _Hash: ...
-def sha1(arg: _DataType = ...) -> _Hash: ...
-def sha224(arg: _DataType = ...) -> _Hash: ...
-def sha256(arg: _DataType = ...) -> _Hash: ...
-def sha384(arg: _DataType = ...) -> _Hash: ...
-def sha512(arg: _DataType = ...) -> _Hash: ...
+def md5(string: _DataType = ...) -> _Hash: ...
+def sha1(string: _DataType = ...) -> _Hash: ...
+def sha224(string: _DataType = ...) -> _Hash: ...
+def sha256(string: _DataType = ...) -> _Hash: ...
+def sha384(string: _DataType = ...) -> _Hash: ...
+def sha512(string: _DataType = ...) -> _Hash: ...
```

Comment:
I just tested this with Python 3.7.5:

```python
>>> hashlib.md5(string="")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: openssl_md5() takes no keyword arguments
```

I suggest to mark them as positional-only arguments.

Reply:
Looks like it changed for py38:
```
>>> sys.version
'3.8.0 (default, Jan  3 2020, 22:27:51) \n[Clang 10.0.1 (clang-1001.0.46.4)]'
>>> hashlib.md5(string='')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: Unicode-objects must be encoded before hashing
```
I'll update the PR.